agent/lib/kontena/logging.rb

```diff
@@ -23,47 +22,31 @@ def logger
     end
 
     # Send a debug message
-    # @param [String] string
+    # @param message [String]
     # @yield optionally set the message using a block
-    def debug(string = nil, &block)
-      if block_given?
-        logger.debug(self.class.name, &block)
-      else
-        logger.debug(self.class.name) { string }
-      end
+    def debug(message = nil, &block)
+      logger.add(Logger::DEBUG, message, self.class.name, &block)
     end
 
     # Send a info message
-    # @param [String] string
+    # @param message [String]
     # @yield optionally set the message using a block
-    def info(string = nil, &block)
-      if block_given?
-        logger.debug(self.class.name, &block)
-      else
-        logger.info(self.class.name) { string }
-      end
+    def info(message = nil, &block)
+      logger.add(Logger::INFO, message, self.class.name, &block)
     end
 
     # Send a warning message
-    # @param [String] string
+    # @param message [String]
```

Comment:
Why order is swapped?

Reply:
You mean the YARD `@param` tag? That's how it's documented now: http://www.rubydoc.info/gems/yard/file/docs/Tags.md#param

>  @param name [Types] description 

Although I'm pretty sure that hasn't always been the case, the examples used to be like `@param [String] message string to print`, and I think both orders work... IMO this new (?) ordering is much better when you actually have a param description.

Reply:
I'd still keep the old ordering when changing existing code, except in this case, I'm changing all the `@param`s for everything in this class/module/file.